ee/ucp/authorization/isolate-nodes.md

```diff
@@ -206,79 +198,47 @@ for Kubernetes workloads.
     apiVersion: v1
     kind: Namespace
     metadata:
-      Name: ops-nodes
+      Name: namespace-name
     ```
-4.  Click **Create** to create the `ops-nodes` namespace.
+4.  Click **Create** to create the `namespace-name` namespace.
 
 ### Grant access to the Kubernetes namespace
 
-Create a grant to the `ops-nodes` namespace for the `Ops` team by following the
-same steps that you used to grant access to the `/Prod` collection, only this
-time, on the **Create Grant** page, pick **Namespaces**, instead of
-**Collections**.
+Create a grant to the `namespace-name` namespace:
+
+1. On the **Create Grant** page, select **Namespaces**.
 
 ![](../images/isolate-nodes-5.png){: .with-border}
 
-Select the **ops-nodes** namespace, and create a `Full Control` grant for the
-`Ops` team.
+Select the **namespace-name** namespace, and create a `Full Control` grant.
```

Comment:
This step should be numbered (#2).

Reply:
A suggestion: Indent the two screenshots so that they align with the start of text following the step numbers.